examples/Examples/BatchTransfer.hs

```diff
@@ -6,11 +6,12 @@ import           Data.Type.Equality                              (type (~))
 
 import           ZkFold.Base.Data.Vector                         (Vector)
 import           ZkFold.Symbolic.Cardano.Contracts.BatchTransfer (Tx, TxOut, batchTransfer)
-import           ZkFold.Symbolic.Cardano.Types                   (Bool, ByteString, F)
+import           ZkFold.Symbolic.Cardano.Types                   (Bool, ByteString)
 import           ZkFold.Symbolic.Class                           (Symbolic (BaseField))
-import           ZkFold.Symbolic.Data.Eq                         (Eq)
+import ZkFold.Base.Algebra.EllipticCurve.BLS12_381 (BLS12_381_Scalar)
+import ZkFold.Base.Algebra.Basic.Field (Zp)
 
-exampleBatchTransfer ::
-  (Symbolic c, BaseField c ~ F, Eq (Bool c) (TxOut c)) =>
-  Tx c -> Vector 5 (TxOut c, TxOut c, ByteString 256 c) -> Bool c
+type F = Zp BLS12_381_Scalar
```

Comment:
This definition of `F` can also be removed together with `BaseField c ~ F` constraint